QUEST_LV_0100.tsv

```diff
@@ -5051,9 +5051,9 @@ QUEST_LV_0100_20150317_005046	The Essence of Light is ready. Talk to Follower Va
 QUEST_LV_0100_20150317_005047	Open the entrance of the chapel
 QUEST_LV_0100_20150317_005048	Vaidutis told you to use the Light Crystal to release the demonic barrier at the chapel gates. Powerful monsters will be attracted to the power of the Light Crystal and may attack you, so he told you to be careful.
 QUEST_LV_0100_20150317_005049	Defeat Mummyghast
-QUEST_LV_0100_20150317_005050	Move to the entrance of Kateen Forest
+QUEST_LV_0100_20150317_005050	Move to Entrance of Kateen Forest
```

Comment:
Did we do this with the Royal Mausoleum as well? Was it:

Move to the Royal Mausoleum?
or
Move to Royal Mausoleum?